### PR TITLE
MINOR: Bump latest 2.6 version to 2.6.2

### PR DESCRIPTION
**Pull request by ableegoldman**

### CODE REVIEW

**Comment by ableegoldman** [comment]
Cherrypicked back to 2.8 and 2.7

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -60,7 +60,7 @@ RUN mkdir -p "/opt/kafka-2.2.2" && chmod a+rw /opt/kafka-2.2.2 && curl -s "$KAFK
 RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.3.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.3.1"
 RUN mkdir -p "/opt/kafka-2.4.1" && chmod a+rw /opt/kafka-2.4.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.4.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.4.1"
 RUN mkdir -p "/opt/kafka-2.5.1" && chmod a+rw /opt/kafka-2.5.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.5.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.5.1"
-RUN mkdir -p "/opt/kafka-2.6.1" && chmod a+rw /opt/kafka-2.6.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.6.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.6.1"
+RUN mkdir -p "/opt/kafka-2.6.2" && chmod a+rw /opt/kafka-2.6.2 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.6.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.6.2"
```
Comment (chia7712) [inline tests/docker/Dockerfile:63]: @ableegoldman Sorry for bothering you. It seems to me the link:https://s3-us-west-2.amazonaws.com/kafka-packages/kafka_2.12-2.6.2.tgz is not available. Have we uploaded related files?

Comment (ableegoldman) [reply tests/docker/Dockerfile:63]: Is it possible you just don't have access to view it? I myself get an `access denied` when I click on that link directly, but if you go to [https://s3-us-west-2.amazonaws.com/kafka-packages/](https://s3-us-west-2.amazonaws.com/kafka-packages/) and search for `kafka_2.12-2.6.2.tgz` it shows up with all the other versions

Comment (ableegoldman) [reply tests/docker/Dockerfile:63]: @chia7712 it should be fixed now. Thanks for bringing this up

Comment (chia7712) [reply tests/docker/Dockerfile:63]: @ableegoldman thanks!

**Pull request merged**